multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/multiprocessing/pool.py", line 121, in worker
    result = (True, func(*args, **kwds))
  File "create_fred.py", line 843, in passiveFredParse
    claim_g=nx.relabel_nodes(claim_g,lambda x:nodelabel_mapper(x))
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/site-packages/networkx/relabel.py", line 93, in relabel_nodes
    m = {n: mapping(n) for n in G}
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/site-packages/networkx/relabel.py", line 93, in <dictcomp>
    m = {n: mapping(n) for n in G}
  File "create_fred.py", line 843, in <lambda>
    claim_g=nx.relabel_nodes(claim_g,lambda x:nodelabel_mapper(x))
  File "create_fred.py", line 599, in nodelabel_mapper
    bool(regex_vn.match(text)):'vn:'+regex_vn.match(text)[1],
TypeError: 'NoneType' object is not subscriptable
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "create_fred.py", line 1121, in <module>
    createFred(args.rdfpath,args.graphpath,args.fcgtype,args.init,args.passive,args.cpu,args.compilefred)
  File "create_fred.py", line 1088, in createFred
    output=sorted([p.get() for p in results],key=lambda x:x[0])
  File "create_fred.py", line 1088, in <listcomp>
    output=sorted([p.get() for p in results],key=lambda x:x[0])
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/multiprocessing/pool.py", line 657, in get
    raise self._value
TypeError: 'NoneType' object is not subscriptable

real	0m9.919s
user	1m58.456s
sys	0m4.736s
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/multiprocessing/pool.py", line 121, in worker
    result = (True, func(*args, **kwds))
  File "create_fred.py", line 843, in passiveFredParse
    claim_g=nx.relabel_nodes(claim_g,lambda x:nodelabel_mapper(x))
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/site-packages/networkx/relabel.py", line 93, in relabel_nodes
    m = {n: mapping(n) for n in G}
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/site-packages/networkx/relabel.py", line 93, in <dictcomp>
    m = {n: mapping(n) for n in G}
  File "create_fred.py", line 843, in <lambda>
    claim_g=nx.relabel_nodes(claim_g,lambda x:nodelabel_mapper(x))
  File "create_fred.py", line 599, in nodelabel_mapper
    bool(regex_vn.match(text)):'vn:'+regex_vn.match(text)[1],
TypeError: 'NoneType' object is not subscriptable
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "create_fred.py", line 1121, in <module>
    createFred(args.rdfpath,args.graphpath,args.fcgtype,args.init,args.passive,args.cpu,args.compilefred)
  File "create_fred.py", line 1088, in createFred
    output=sorted([p.get() for p in results],key=lambda x:x[0])
  File "create_fred.py", line 1088, in <listcomp>
    output=sorted([p.get() for p in results],key=lambda x:x[0])
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/multiprocessing/pool.py", line 657, in get
    raise self._value
TypeError: 'NoneType' object is not subscriptable

real	0m7.505s
user	2m5.736s
sys	0m4.660s
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/multiprocessing/pool.py", line 121, in worker
    result = (True, func(*args, **kwds))
  File "create_fred.py", line 950, in compileClaimGraph1
    saveClaimGraph(claim_g,filename,1)
  File "create_fred.py", line 907, in saveClaimGraph
    claim_g=nx.relabel_nodes(claim_g,lambda x:nodelabel_mapper(x))
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/site-packages/networkx/relabel.py", line 93, in relabel_nodes
    m = {n: mapping(n) for n in G}
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/site-packages/networkx/relabel.py", line 93, in <dictcomp>
    m = {n: mapping(n) for n in G}
  File "create_fred.py", line 907, in <lambda>
    claim_g=nx.relabel_nodes(claim_g,lambda x:nodelabel_mapper(x))
  File "create_fred.py", line 600, in nodelabel_mapper
    bool(regex_dbpedia.match(text)):'db:'+regex_dbpedia.match(text)[1],
TypeError: 'NoneType' object is not subscriptable
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "create_fred.py", line 1121, in <module>
    createFred(args.rdfpath,args.graphpath,args.fcgtype,args.init,args.passive,args.cpu,args.compilefred)
  File "create_fred.py", line 1070, in createFred
    output=sorted([p.get() for p in results],key=lambda x:x[0])
  File "create_fred.py", line 1070, in <listcomp>
    output=sorted([p.get() for p in results],key=lambda x:x[0])
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/multiprocessing/pool.py", line 657, in get
    raise self._value
TypeError: 'NoneType' object is not subscriptable

real	0m4.921s
user	0m2.032s
sys	0m2.140s
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/multiprocessing/pool.py", line 121, in worker
    result = (True, func(*args, **kwds))
  File "create_fred.py", line 950, in compileClaimGraph1
    saveClaimGraph(claim_g,filename,1)
  File "create_fred.py", line 907, in saveClaimGraph
    claim_g=nx.relabel_nodes(claim_g,lambda x:nodelabel_mapper(x))
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/site-packages/networkx/relabel.py", line 93, in relabel_nodes
    m = {n: mapping(n) for n in G}
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/site-packages/networkx/relabel.py", line 93, in <dictcomp>
    m = {n: mapping(n) for n in G}
  File "create_fred.py", line 907, in <lambda>
    claim_g=nx.relabel_nodes(claim_g,lambda x:nodelabel_mapper(x))
  File "create_fred.py", line 599, in nodelabel_mapper
    bool(regex_vn.match(text)):'vn:'+regex_vn.match(text)[1],
TypeError: 'NoneType' object is not subscriptable
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "create_fred.py", line 1121, in <module>
    createFred(args.rdfpath,args.graphpath,args.fcgtype,args.init,args.passive,args.cpu,args.compilefred)
  File "create_fred.py", line 1070, in createFred
    output=sorted([p.get() for p in results],key=lambda x:x[0])
  File "create_fred.py", line 1070, in <listcomp>
    output=sorted([p.get() for p in results],key=lambda x:x[0])
  File "/N/u/zkachwal/Carbonate/miniconda3/lib/python3.7/multiprocessing/pool.py", line 657, in get
    raise self._value
TypeError: 'NoneType' object is not subscriptable

real	0m5.156s
user	0m2.680s
sys	0m2.308s

real	0m9.211s
user	0m5.584s
sys	0m1.424s
